podvm/Makefile.inc

```diff
@@ -15,6 +15,8 @@ IMAGE_PREFIX := podvm
 
 HOST_ARCH   := $(shell uname -m)
 ARCH        ?= $(HOST_ARCH)
+# Normalise x86_64 / amd64 for input ARCH
+ARCH        := $(subst amd64,x86_64,$(ARCH))
```

Comment:
ARCH is also used as ARG in the Dockerfile. If it's passed as container build arg, then ARCH will be set first in L17 and then modified. But still want to re-confirm from you :-) .. Do you see any issues ?

Reply:
It matching with the ARG in the Dockerfile is on purpose. L17 won't set it again if it is set as a build-arg in the docker build. 
L19 means that we can specify `amd64` `x86_64` for the build-arg and it will still work as expected.